src/vivarium_testing_utils/automated_validation/data_loader.py

```diff
@@ -84,6 +84,21 @@ def _load_artifact(results_dir: str) -> Artifact:
     def _load_from_artifact(self, dataset_key: str) -> pd.DataFrame:
         data = self._artifact.load(dataset_key)
         self._artifact.clear_cache()
+        # if data has value columns of format draw_1, draw_2, etc., drop the draw_ prefix
+        # and melt the data into long format
+        if data.columns.str.startswith("draw_").all():
```

Comment:
Having every single col (not index) start with "draw_" is an artifact contract, correct?